src/Adapters/EventSource.Netstandard13/EventSourceTelemetryModule.cs

```diff
@@ -154,9 +154,9 @@ protected override void OnEventSourceCreated(EventSource eventSource)
         private void EnableAsNecessary(EventSource eventSource)
         {
             // Special case: enable TPL activity flow for better tracing of nested activities.
-            if (eventSource.Guid == EventSourceTelemetryModule.TplEventSourceGuid)
+            if (eventSource.Guid == TplActivities.TplEventSourceGuid)
             {
-                this.EnableEvents(eventSource, EventLevel.LogAlways, (EventKeywords)EventSourceTelemetryModule.TaskFlowActivityIds);
+                this.EnableEvents(eventSource, EventLevel.LogAlways, (EventKeywords) TplActivities.TaskFlowActivityIdsKeyword);
```

Comment:
I heard there is a problem with hierarchical ids on framework 46 in eventsource. Are you sure enabling those will make any good for event source listener?